src/components/Filter/FilterInput.vue

```diff
@@ -14,9 +14,9 @@
     role="search"
     tabindex="0"
     :aria-labelledby="searchAriaLabelledBy"
-    :class="{ 'focus': showSuggestedTags }"
+    :class="{ 'focus': showSuggestedTags && !preventFocusStyle }"
```

Comment:
@marinaaisa this would work in this case, but what about use cases where you want to keep the focus even when there are no tags? I think that we are tying up two unrelated things (the existence of tags with the focus logic)

Reply:
Ended up removing the prop and leaving the focus style on the element ([bedd4e0](https://github.com/apple/swift-docc-render/pull/396/commits/bedd4e06f51824468835b932b2a9640aad3afbd3))

Reply:
Nice! I think it's better in terms of AX :)